src/components/SearchFilter/SearchFilter.js

```diff
@@ -0,0 +1,131 @@
+import React, { Component } from 'react';
+import withStyles from '@material-ui/core/styles/withStyles';
+import Typography from '@material-ui/core/Typography';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import { Checkbox } from '@material-ui/core';
+import { CheckBox, CheckBoxOutlineBlank } from '@material-ui/icons';
```

Comment:
2 x checkbox, misschien wat meer onderscheid maken

Reply:
I agree. Usually, I suffix the icons with `Icon` e.g. `CheckBoxIcon`